.storybook/preview.js

```diff
@@ -21,9 +20,11 @@ Array
     };
   });
 
-const availableLanguages = Object
-  .entries(getAvailableLanguages())
-  .map(([title, value]) => ({ value, title }));
+const availableLanguages = [
+  {value: 'en', title: 'English'},
+  {value: 'fr', title: 'Français'},
+  {value: 'missing', title:'🤬 Missing'},
+];
```

Comment:
nitpick: prettier adds spaces at the beginning and the end of object declarations

:arrow_right: this change is available in my proposition branch: https://github.com/CleverCloud/clever-components/tree/typecheck-i18n--hubert